elastic/kibana
# fix Canvas can not render Map with query or filter in 8.15

Resolves https://github.com/elastic/kibana/issues/199318. Issue only exists in 8.15 branch.

### Test steps
1) install sample web logs
2) create new canvas work pad
3) Click "Add from library" and add "[Logs] Total Requests and Bytes" map to workpad
4) Click panel actions button and select "Edit map"
5) Add query "bytes > 5000" to map and click "Save and return".
6) Ensure map panel is rendered with filter notification popover
    ![Image](https://github.com/user-attachments/assets/2689bbb5-c22b-4c28-9e86-2eeef48f2676)
